doc/whatsnew/2.13.rst

```diff
@@ -575,3 +575,7 @@ Other Changes
   separated (e.g. by parens).
 
   Closes #5769
+
+* Fix crash in ``super-init-not-called`` checker when using ``ctypes.Union``.
+
+  Closes #6027
```

Comment:
Per https://github.com/PyCQA/pylint/pull/6040#discussion_r837901916 I think we're not adding hotfixes to the whatsnew -- just the changelog.

```suggestion
```